build.gradle

```diff
@@ -60,3 +60,7 @@ tasks.create("currentVersion") {
         println version
     }
 }
+
+/*tasks.getByName("runIde").configure {
+    systemProperty("TABNINE_LOG_FILE_PATH", "/home/amir/t9-ij.log")
+}*/
```

Comment:
Dont leave commented code

Reply:
This is the example of how to use this parameter in debug. 
It is not immediately obvious.
I'll add a comment explaining

Reply:
Actually, I did something different.

I added a `logFilePath` gradle property that sets this value, so that you can add `-PlogFilePath={path)` when running form gradle to set it

Reply:
@amircodota maybe worth documenting this somewhere?

Reply:
where?